### ISSUE TITLE
add transparency control map-page

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#175]
Based on e.g. on https://github.com/dayjournal/Leaflet.Control.Opacity.
Only on the modeled layer. 

Default settings:
```
collapsed: true,
position: 'bottomleft',
label: "Transparency" (language specific)
```

**Comment by janschulte** [comment]
Done with: 2f36062a08d48fd8ac612bd3715f6d1ebaac9c92

**Comment by opeeters** [comment]
Three issues:
- only the modeled layer should go into the opacity control
- The label should be hidden when the control is in a collapsed state (if this is impossible I guess we should remove the label all together)
- The layer names should not be shown in an expanded state

Maybe we are using the wrong plugin. Maybe have a look at this one:
https://github.com/lizardtechblog/Leaflet.OpacityControls
The `opacitySlider` seems OK

**Comment by opeeters** [comment]
One new problem. When changing the language setting of the app, the transparency layer icon is drawn multiple times. See screenshot:

![Screenshot_20190826-122315](https://user-images.githubusercontent.com/323719/63684289-cefec580-c7fc-11e9-828c-9f3d1f3e3294.png)

Steps to reproduce:
- open app
- change to map view
- change language in app settings

**Comment by opeeters** [comment]
after update packages, not reproducible..
closing issue

**Comment by opeeters** [comment]
After adjusting the transparency for one pollutant (or on the BelAQI tab) and switching to another, the transparency is not being set anymore.
Steps to reproduce:
- open BelAir
- go to the map page
- adjust the transparency of the AQI layer
- switch to one of the pollutants and try to adjust transparency
Result:
- you can move the slider, but the transparency remains at the default

Actually, it would also be better to take the set transparency from one pollutant tab to the next and only restore the default transparency when the map page is reloaded

**Comment by janschulte** [comment]
Fixed with aaf6b10dea79a05c4cf3754deb16b5ec219d13cc

The transparency will now also bei reset, after switching to another pollutant.
